de.metas.salescandidate.base/src/main/java/de/metas/ordercandidate/api/OLCandFactory.java

```diff
@@ -43,7 +47,10 @@ public OLCand toOLCand(@NonNull final I_C_OLCand record)
 				.pricingSystemId(PricingSystemId.ofRepoIdOrNull(record.getM_PricingSystem_ID()))
 				.deliveryRule(DeliveryRule.ofNullableCode(record.getDeliveryRule()))
 				.deliveryViaRule(DeliveryViaRule.ofNullableCode(record.getDeliveryViaRule()))
-				.shipperId(null)
+				.shipperId(ShipperId.ofRepoIdOrNull(record.getM_Shipper_ID()))
+				.paymentRule(PaymentRule.ofCode(record.getPaymentRule()))
```

Comment:
quickly checked in I_C_OLCand: paymentRule is not mandatory, so it might be `null`

Reply:
Used ofNullableCode instead